libs/ui/icon/helm/src/lib/hlm-icon.component.ts

```diff
@@ -1,24 +1,49 @@
-import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation, signal } from '@angular/core';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+  effect,
+  signal,
+} from '@angular/core';
 import { IconName, NgIconComponent } from '@ng-icons/core';
+import { hlm } from '@ng-spartan/ui/core/helm';
+import { VariantProps, cva } from 'class-variance-authority';
 import { ClassValue } from 'clsx';
 
+const iconVariants = cva('inline-flex', {
+  variants: {
+    variant: {
+      xSmall: 'h-3 w-3',
```

Comment:
Can we name this the same as tailwind does: xs,sm,base,lg,xl and maybe add a option (could be the default) that does not add any tw classes so the user can set their own width and height?

Reply:
Great idea! :) I added/changed the suggested sizes & added support for passing classes directly to ngIcon as well as using ngIcon sizes.